config/monitoring/cluster_monitoring_config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-monitoring
```

Comment:
It's going to be a problem in OSD.
I don't think we can't create anything in openshift-* namespaces in OSD.
So, I guess user workload monitoring is not enableable in OSD? Worth checking with app-sre and/or platform-sre folks.

Reply:
yeah, we need to check that indeed 🤯